pages/index.js

```diff
@@ -1,24 +1,10 @@
 import Form from '../components/Form';
-import { useState } from 'react';
 
 export default function Home({ coins }) {
-  let [value, setValue] = useState();
-
-  function calculatePrice(value, expense) {
-    const crypto = coins[0];
-    const priceCrypto = crypto.current_price;
-    console.log(crypto.current_price);
-    let result = 0;
-    if (expense) {
-      result = ((value * priceCrypto) / expense).toFixed(0);
-    }
-    setValue(result);
-  }
-
   return (
     <>
       <div>
-        <Form onCalculatePrice={calculatePrice} value={value} />
+        <Form coins={coins} />
       </div>
     </>
```

Comment:
The outer fragment is not needed, you have the `div` here as a wrapper.

Reply:
Yes ok I just see it too. I have sorted out my index. Before I had brought it. I have changed it. Thank you.